tests/test_image_vec_rep_resnet.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from core.feluda import Feluda
+from core.models.media_factory import ImageFactory
+
+
+class Test(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Intializing Feluda
+        cls.feluda = Feluda('image_vec_rep_resnet-config.yaml')
+        cls.feluda.setup()
+
+    @classmethod
+    def tearDownClass(cls):
+        # Clean up resources if needed
+        pass
+
+    @skip
```

Comment:
hi @plon-Susk7 one small last change is needed, you forgot to import skip
```py
from unittest.case import skip
```
the test will fail otherwise

Reply:
Really sorry for overlooking this. I've made the changes.

Reply:
no no all good, things look good now!

will test once locally and merge the PR!